eng/testing/tests.mobile.targets

```diff
@@ -10,7 +10,7 @@
     <PublishingTestsRun>true</PublishingTestsRun>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetOS)' == 'Browser' And '$(UseDefaultBlazorWASMFeatureSwitches)' != 'false'">
+  <PropertyGroup Condition="'$(TargetOS)' == 'Browser' And '$(UseDefaultBlazorWASMFeatureSwitches)' == 'true'">
```

Comment:
Do you need to set these flags to true by default if they are not set?

Reply:
I'm not sure if I need to activate those flags b/c the current state is to switch them off by default. @steveisok

Reply:
I *think* we'll want to go with the old way.  @radical does that make sense to you?

Reply:
@MaximLipnin The tests are all erroring for Browser wasm because of this line. This condition `'$(UseDefaultBlazorWASMFeatureSwitches)' == 'true'` is not satisfied in the lane, and thus `RunTests.sh` contains just `-notrait category=OuterLoop -notrait category=failing $RSP_FILE` instead of the usual L15 or L16 below. Hence `./RunTests.sh: line 29: -notrait: command not found` in the error logs.

We can default the projects to have `UseDefaultBlazorWASMFeatureSwitches` be true as suggested prior by Santiago.

Reply:
Sorry, but I don't get what is the difference between these two cases:
1. '$(UseDefaultBlazorWASMFeatureSwitches)' != 'false' when UseDefaultBlazorWASMFeatureSwitches is set to false;
2. '$(UseDefaultBlazorWASMFeatureSwitches)' == 'true' when UseDefaultBlazorWASMFeatureSwitches is not set.

Reply:
I think this is related to what I was saying before on my concerns. When you had `$(UseDefaultBlazorWASMFeatureSwitches)` != 'false'` there was no one setting that flag to any value other than the linker tests which was setting it to false. 

The regular tests (runtime lane) was not setting it to any value, so the condition was satisfied as it would be evaluated as `'' != 'false'` which is true. Now that you changed the condition to `== 'true'` it is being evaluated to `'' == 'true'` which is false.

That is what I meant with "Do you need to set these flags to true by default if they are not set?" when the flag is not set to anything because before this change they were turned on by default as an empty value (not setting it) meant that they would be satisfied in the conditions.

Reply:
Right, both of those scenarios are the same in that scope, but for all other projects that use `tests.mobile.targets` if `UseDefaultBlazorWASMFeatureSwitches` is not set, aka `UseDefaultBlazorWASMFeatureSwitches=''`, then (1) is true whereas (2) is false.

I'm not familiar with `eng/testing/linker/SupportFiles/Directory.Build.props`, but will those properties be loaded in the currently failing lane `Build Browser wasm Release AllSubsets_Mono`? If not, then that would be why (1) was passing before the change, whereas now (2) is failing. Do you happen to know if the tests were passing back when (1) was in place?

Reply:
Thanks, I completely missed the fact that `UseDefaultBlazorWASMFeatureSwitches` had `false` default value only for the linker tests 🤯

Reply:
Set the `UseDefault*FeatureSwitches` properties to true in case of empty value.

Reply:
Yes, exactly. Sorry for not elaborating since the beginning, that is what I meant initially. So once you add the default value to true if empty you will need to revert the change to set the switches to false in the linker tests.

Reply:
Thanks, moved back the switches' default values for the linker tests.